### ISSUE TITLE
virtualbox shared folder options wrongly used

### ISSUE THREAD

**Issue by vpereira** [mitchellh/vagrant#7662]
I'm running my guest on opensuse, where we have user vagrant, group users. So as synced_folder parameters I must set the parameters as described here: https://github.com/mitchellh/vagrant/blob/f8439869e7003255593093d3c7a2e87695368118/website/source/docs/synced-folders/basic_usage.html.md

I set the owner and group in my Vagrantfile as

````
  config.vm.synced_folder ".", "/vagrant", group:"users", owner: "vagrant"
````

and then the output from vagrant up looks like:

````
  set -e
  # looks good
  mount -t vboxsf -o uid=`id -u vagrant`,gid=`getent group users | cut -d: -f3` vagrant /vagrant
  # its wrong, here gid should be gid=`id -g vagrant`
  # since its asking the gid from the user. gid from the group doesnt make sense
  mount -t vboxsf -o uid=`id -u vagrant`,gid=`id -g users` vagrant /vagrant
````

looking the source code in:

https://github.com/mitchellh/vagrant/blob/1277f192514910f01ecd71d9acd5383867ce3372/plugins/guests/linux/cap/mount_virtualbox_shared_folder.rb#L27

it should be:

````
mount_gid_old = "`id -g #{options[:owner]}
````
right?

**Comment by vpereira** [comment]
funny thing is that with the ````set -e```` vagrant shouldn't exit with one of both commands fail. However for me,  if one of both commands fail, vagrant aborts. I'm able to ````vagrant ssh```` it and see my sync folder mounted. However because of the abortion the subsequent events (i.e provisioning) doesn't happens.. any idea?

**Comment by chrisroberts** [comment]
Fixed via #7720. Thanks for the report!
